cabal.project

```diff
@@ -153,181 +167,209 @@ source-repository-package
+  --sha256: 10q7n4k1c4yp5vkbwxgvx4ssljiw7q8ai4jn69k3yrmf5sxipawy
   subdir: ouroboros-consensus/ouroboros-consensus-test-infra
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: e71aa72adc7e876cda4ff19b5cc0900167fa9804
-  --sha256: 0a16fd350kvyq4a23y8ma09r5sfs4xsr0gll0ssndd6ygpsd5s4b
+  tag: 11e5609e01f4703870e88b2712e2c14da0338b65
+  --sha256: 10q7n4k1c4yp5vkbwxgvx4ssljiw7q8ai4jn69k3yrmf5sxipawy
   subdir: ouroboros-consensus-byron
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: e71aa72adc7e876cda4ff19b5cc0900167fa9804
-  --sha256: 0a16fd350kvyq4a23y8ma09r5sfs4xsr0gll0ssndd6ygpsd5s4b
+  tag: 11e5609e01f4703870e88b2712e2c14da0338b65
+  --sha256: 10q7n4k1c4yp5vkbwxgvx4ssljiw7q8ai4jn69k3yrmf5sxipawy
   subdir: ouroboros-consensus-byronspec
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: e71aa72adc7e876cda4ff19b5cc0900167fa9804
-  --sha256: 0a16fd350kvyq4a23y8ma09r5sfs4xsr0gll0ssndd6ygpsd5s4b
+  tag: 11e5609e01f4703870e88b2712e2c14da0338b65
+  --sha256: 10q7n4k1c4yp5vkbwxgvx4ssljiw7q8ai4jn69k3yrmf5sxipawy
   subdir: ouroboros-consensus-shelley
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: e71aa72adc7e876cda4ff19b5cc0900167fa9804
-  --sha256: 0a16fd350kvyq4a23y8ma09r5sfs4xsr0gll0ssndd6ygpsd5s4b
+  tag: 11e5609e01f4703870e88b2712e2c14da0338b65
+  --sha256: 10q7n4k1c4yp5vkbwxgvx4ssljiw7q8ai4jn69k3yrmf5sxipawy
   subdir: ouroboros-consensus-cardano
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: e71aa72adc7e876cda4ff19b5cc0900167fa9804
-  --sha256: 0a16fd350kvyq4a23y8ma09r5sfs4xsr0gll0ssndd6ygpsd5s4b
+  tag: 11e5609e01f4703870e88b2712e2c14da0338b65
+  --sha256: 10q7n4k1c4yp5vkbwxgvx4ssljiw7q8ai4jn69k3yrmf5sxipawy
   subdir: typed-protocols
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: e71aa72adc7e876cda4ff19b5cc0900167fa9804
-  --sha256: 0a16fd350kvyq4a23y8ma09r5sfs4xsr0gll0ssndd6ygpsd5s4b
+  tag: 11e5609e01f4703870e88b2712e2c14da0338b65
+  --sha256: 10q7n4k1c4yp5vkbwxgvx4ssljiw7q8ai4jn69k3yrmf5sxipawy
   subdir: typed-protocols-examples
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-node
-  tag: b5b0a293a890af48070d0ea5df97dd48dc0551e7
-  --sha256: 1dp0lb4cy9yq9k3d55gk0dlmkhpjzfyw9qk9cvcw95q50c5jg58s
+  tag: 8c72fc00cb75b7e7a2c9423ba50c639f6236c88b
+  --sha256: 0h08p1plyxik8wwcws2cvgkvlxlhgg33v3mfj110lzrfiwq4yygs
+  subdir: cardano-api
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-node
+  tag: 8c72fc00cb75b7e7a2c9423ba50c639f6236c88b
+  --sha256: 0h08p1plyxik8wwcws2cvgkvlxlhgg33v3mfj110lzrfiwq4yygs
   subdir: cardano-config
```

Comment:
Note that I haven't yet updated to the latest revision of `cardano-node` due to the fact that it would also require some changes in `cardano-db-sync`. 

The tag that I've used, `8c72fc00cb75b7e7a2c9423ba50c639f6236c88b`, is still relatively recent (from 2 days ago) and provides all of the `cardano-api` utilities that we need.

Reply:
I need the following as well to build locally:

```

source-repository-package
  type: git
  location: https://github.com/input-output-hk/cardano-node
  tag: 8c72fc00cb75b7e7a2c9423ba50c639f6236c88b
  --sha256: 0h08p1plyxik8wwcws2cvgkvlxlhgg33v3mfj110lzrfiwq4yygs
  subdir: cardano-cli
```

Reply:
I'm also needing:

```
package small-steps
  tests: False
```

Reply:
I recently force-pushed to remove a `cardano-cli` dependency that I mistakenly added to `cardano-submit-api`. Mind trying again? It should build fine without that `source-repository-package` for `cardano-cli`.